lib/chef-dk/command/verify.rb

```diff
@@ -96,6 +96,20 @@ def components
         c.smoke_test { run_in_tmpdir("kitchen init") }
       end
 
+      # We only ship delivery-cli on *nix only ATM
+      unless Chef::Platform.windows?
+        add_component "delivery-cli" do |c|
+          # There is no base_dir as delivery-cli is a single binary in
+          # `/opt/chefdk/bin`.
+          c.base_dir = "chef-dk"
+          c.smoke_test do
+            tmpdir do |cwd|
+              sh!("delivery setup --user=shipit --server=delivery.shipit.io --ent=chef --org=squirrels", cwd: cwd)
```

Comment:
Interested in the answer here, as I could see this command including some kind of connectivity check in the future. But if this the best we can do right now, I'm cool with it.